package/MDAnalysis/core/universe.py

```diff
@@ -170,62 +229,65 @@ class Universe(object):
 
     Parameters
     ----------
-    topology : str, Topology object or stream
-        A CHARMM/XPLOR PSF topology file, PDB file or Gromacs GRO file; used to
-        define the list of atoms. If the file includes bond information,
-        partial charges, atom masses, ... then these data will be available to
-        MDAnalysis. A "structure" file (PSF, PDB or GRO, in the sense of a
-        topology) is always required. Alternatively, an existing
-        :class:`MDAnalysis.core.topology.Topology` instance may also be given.
-    topology_format
+    topology : `~MDAnalysis.core.topology.Topology`, str, stream, `np.ndarray`, None
+        The topology defines the list of atoms. This can be provided 
+        as filenames, streams of filenames, numpy arrays, or None for 
+        an empty universe.
+    coordinates : str, stream, list of str, list of stream (optional)
+        Coordinates can be provided as streams of filenames either of 
```

Comment:
Same here, reword so that the possibility of giving a path to the users' single coordinate file is the first in list of options

Reply:
What do you think of it now? I brought back some of the old wording